publisher-sdk/src/main/java/com/criteo/publisher/logging/Logger.java

```diff
@@ -34,7 +34,7 @@ public Logger(
       @NonNull Class<?> klass,
       @NonNull BuildConfigWrapper buildConfigWrapper
   ) {
-    this.tag = klass.getSimpleName();
+    this.tag = "Crto" + klass.getSimpleName();
```

Comment:
Why not `Criteo`? :P

Reply:
The `Criteo` word might appear in other cases. For instance when using mediation adapters, AdMob and MoPub are printing stuff about Criteo classes.

But moreover, this is to save few chars because of `IllegalArgumentException` that could be thrown on API <= 23 if length of tag > 23: https://developer.android.com/reference/android/util/Log.html#isLoggable(java.lang.String,%20int)